smdebug/xgboost/collection.py

```diff
@@ -14,6 +14,7 @@ def _register_default_collections(self):
             self.create_collection(c)
         self.get(CollectionKeys.HYPERPARAMETERS).include("^hyperparameters/.*$")
         self.get(CollectionKeys.METRICS).include("^[a-zA-z]+-[a-zA-z0-9]+$")
+        self.get(CollectionKeys.LOSSES).include("^[a-zA-z]+-[a-zA-z0-9]+$")
```

Comment:
I just have a minor comment: if we make the loss collection the same like metrics collection, then it can include metrics such as error, rmse, mae, mape (which is expected to decrease during training) and metrics such as aoc, map (which is expected to increase during training). So the loss not decreasing rule could potentially create false positives.

Reply:
The name of the metric can change across training jobs, right?
Maybe we should update this based on hyperparameter?